Update list libboost* versions that might be sought for and force for
shared object libs instead of static lib.--- a/cmake/FindBoost.cmake
+++ b/cmake/FindBoost.cmake
@@ -312,6 +312,13 @@
   )
 endmacro(_Boost_ADJUST_LIB_VARS)
 
+
+# ------------------------------------------------------------------------------
+#
+# Force the shared library object use instead of the static lib.
+
+set(Boost_USE_STATIC_LIBS        OFF)
+
 #-------------------------------------------------------------------------------
 
 #
@@ -392,7 +399,7 @@
   # The user has not requested an exact version.  Among known
   # versions, find those that are acceptable to the user request.
   set(_Boost_KNOWN_VERSIONS ${Boost_ADDITIONAL_VERSIONS}
-    "1.46.1"
+    "1.49.0" "1.49" "1.48.0" "1.48" "1.46.1"
     "1.46.0" "1.46" "1.45.0" "1.45" "1.44.0" "1.44" "1.43.0" "1.43" "1.42.0" "1.42"
     "1.41.0" "1.41" "1.40.0" "1.40" "1.39.0" "1.39" "1.38.0" "1.38" "1.37.0" "1.37"
     "1.36.1" "1.36.0" "1.36" "1.35.1" "1.35.0" "1.35" "1.34.1" "1.34.0"
